### PR TITLE
CASMTRIAGE-3173 for release/1.2

### PR DESCRIPTION
**Pull request by alexanderkingh**
## Summary and Scope

Docs change for the operations/boot_orchestration/Edit_the_iPXE_Embedded_Boot_Script.md and install/pxe_boot_troubleshooting.md. same as [PR #1552](https://github.com/Cray-HPE/docs-csm/pull/1552) but for reelase/1.2

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [issue id](issue link)
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by mharding-hpe — APPROVED** [review]

File: operations/boot_orchestration/Edit_the_iPXE_Embedded_Boot_Script.md [hunk 1]
```diff
@@ -39,52 +39,54 @@ This procedure requires administrative privileges.
     ifclose net0 || echo No routes to drop.
     ```
 
-    -   **Option 1:** Edit the cray-ipxe-bss-ipxe ConfigMap directly.
+<<<<<<< HEAD
```
Comment (mharding-hpe) [inline operations/boot_orchestration/Edit_the_iPXE_Embedded_Boot_Script.md:42]: This should not be in there.

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

File: operations/boot_orchestration/Edit_the_iPXE_Embedded_Boot_Script.md (hunk 1 above)
Comment (denniswalker) [reply operations/boot_orchestration/Edit_the_iPXE_Embedded_Boot_Script.md]: Resolved

**Comment by mharding-hpe** [comment]
This should be reverted.

**Pull request merged**